lib/pmdtester/parsers/options.rb

```diff
@@ -0,0 +1,55 @@
+require 'slop'
+
+module PmdTester
+  class Options
+
+    VERSION = '1.0.0'
```

Comment:
Is it possible to add a version suffix, to indicate that this version is work in progress - like "`VERSION = '1.0.0-SNAPSHOT'`. Using 1.0.0 might otherwise be misinterpreted as "finished" and "feature complete"...